lib/task-data/tasks/install-ubuntu.js

```diff
@@ -17,7 +17,15 @@ module.exports = {
         completionUri: 'renasar-ansible.pub',
         version: 'trusty',
         repo: '{{api.server}}/ubuntu',
-        rootPassword: "RackHDRocks!"
+        netImage: '{{api.server}}/ubuntu/install/filesystem.squashfs',
+        rootPassword: "RackHDRocks!",
+        rootSshKey: null,
```

Comment:
The task-schema is relative new feature and now only have only been enabled for majority of install os tasks and obm control tasks, and we still lack a document to describe how to use and define the task-schema.  So all your reference document is the code itself at present. Of course, you are welcome if you have any question about task-schema, either code or unit-test.